### ISSUE TITLE
MAX Request - Button to Remove Behavior

### ISSUE THREAD

**Issue by ArgentArts** [TheGameCreators/GameGuruRepo#1550]
Hello. After adding a Behavior to an entity, I don't see any way to remove it if the end-user decides they no longer want a Behavior attached to that object. Therefore, I would like to request a Remove Behavior button added to the right-side panel within the added Behavior. So, if I added the Door Behavior, as an example, and then later decide that I don't want to use that, I could open that Behavior on the right-side panel and click a big "X" (or whatever) to detach the Behavior from the model and remove it.

**Comment by rvanner** [comment]
@ArgentArts Does this not sort out this issue;

![image](https://user-images.githubusercontent.com/1503593/141675479-18dc17db-df87-45bd-b114-3720712727dd.png)

**Comment by Retrogamebloke** [comment]
Beat me to it! Defo needed, you can't even type in default.lua anymore to remove a script.

**Comment by LeeBamberTGC** [comment]
@rvanner Should we rename DEFAULT to NO BEHAVIOR so users know to select this is no behavior is desired?

**Comment by rvanner** [comment]
@LeeBamberTGC Yes, that would make sense.

**Comment by LeeBamberTGC** [comment]
@rvanner Sorted for next weeks build.

**Comment by ArgentArts** [comment]
Yes, it does. But ... why not just have a big, fat "X" in one of the corners of the Behavior as it appears on the right-side panel. That way, the end-user doesn't have to access look through the actual behaviors to find the one required that removes behaviors. Instead, it would be as simple as selecting the entity and clicking the "X" on the behavior.

**Comment by rvanner** [comment]
@ArgentArts For now we are rounding things off for the Early Access release. There's more to do with Behaviours so we're sure to improve how it all works in time.
